### ISSUE TITLE
Callsite

### ISSUE THREAD

**Issue by mikecg** [uhaciogullari/NLog.Interface#8]
Thanks for this library!

I noticed that when I try to emit callsite information in the log, the log messages end up writing NLog.Interface.LoggerAdapter.Debug instead of the actual site the log was written.

This post discusses the reason and the fix looks very straightforward:
http://stackoverflow.com/questions/5132759/nlog-callsite-information

Are you able to fix this?

**Comment by uhaciogullari** [comment]
Could you please send me a pull request?

**Comment by mikecg** [comment]
Hi,
Does that mean that I should make the changes and commit them?
Thanks,Mike

**Comment by uhaciogullari** [comment]
Yes, fork the repo. Make the changes, commit and send me a pull request. You can read more about it here: https://help.github.com/articles/using-pull-requests/
